### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by jekabolt**

### CODE REVIEW

File: minting-dapp-admin/generate-proto.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (jekabolt) [reply minting-dapp-admin/generate-proto.sh:1]: 5. generate proto

**Comment by dogfrogfog** [comment]
add grpc-http files + generation of this files + add command to `makefile`

**Comment by dogfrogfog** [comment]
i think we can add `go get go.einride.tech/protoc-gen-typescript-http` to makefile to not make users explicitly install package, but i didnt manage to ..... help pls

**Review by dogfrogfog — COMMENTED** [review]
LGTM!

**Review by dogfrogfog — APPROVED** [review]

**Comment by dogfrogfog** [comment]
@jekabolt

File: minting-dapp-admin/Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+REGISTRY=dogfrogfog
+VERSION=dev-admin
+
+install: ## Install the web dependencies
+	yarn install --ignore-engines
+
+run-local: ## Run the local dev server
+	yarn dev
+
+build-local: ## Build dev version
```
Comment (jekabolt) [inline minting-dapp-admin/Makefile:10]: build-local: generate-proto
 generate proto on every build

File: minting-dapp-admin/src/api/index.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+// copy of type inside generated file (no export, need to define explicitly)
+type RequestType = {
+  path: string;
+  method: string;
+  body: string | null;
+};
+
+const getAuthHeaders = (authToken: string) => ({
+  'Grpc-Metadata-Authorization': `Bearer ${authToken}`,
```
Comment (jekabolt) [inline minting-dapp-admin/src/api/index.ts:23]: take authToken from local storage

File: minting-dapp-admin/src/api/index.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+});
+
+export function login(password: string): Promise<LoginResponse> {
+  const authClient = createAuthClient(({ path, body }: RequestType): Promise<LoginResponse> => {
+    return axios
+      .post<LoginRequest, AxiosResponse<LoginResponse>>(path, body && JSON.parse(body))
+      .then((response) => response.data);
+  });
+
+  return authClient.Login({ password });
```
Comment (jekabolt) [inline minting-dapp-admin/src/api/index.ts:33]: put to local storage

File: minting-dapp-admin/webpack.dev.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+	devtool: 'source-map',
+	devServer: {
+		static: {
+			directory: path.join(__dirname, 'public/'),
+		},
+		compress: true,
+		port: 4040,
+		proxy: {
+			'/api': {
+				target: 'http://localhost:4040',
```
Comment (jekabolt) [inline minting-dapp-admin/webpack.dev.js:14]: make it settable from env var if var is empty use default

**Review by dogfrogfog — APPROVED** [review]

**Pull request merged**